CHANGELOG.txt

```diff
@@ -1,3 +1,134 @@
+- [AIRFLOW-5508] Add config setting to limit which StatsD metrics are emitted (#6130)
+- [AIRFLOW-4222] Add cli autocomplete for bash & zsh (#5789)
+- [AIRFLOW-3871] Operators template fields can now render fields inside objects (#4743)
+
+Improvements
+""""""""""""
+- [AIRFLOW-5127] Gzip support for CassandraToGoogleCloudStorageOperator (#5738)
+- [AIRFLOW-5125] Add gzip support for AdlsToGoogleCloudStorageOperator (#5737)
+- [AIRFLOW-5124] Add gzip support for S3ToGoogleCloudStorageOperator (#5736)
+- [AIRFLOW-5653] Log AirflowSkipException in task instance log to make it clearer why tasks might be skipped (#6330)
+- [AIRFLOW-5343] Remove legacy SQLAlchmey pessimistic pool disconnect handling (#6034)
+- [AIRFLOW-5561] Relax httplib2 version required for gcp extra (#6194)
+- [AIRFLOW-5657] Update the upper bound for dill dependency (#6334)
+- [AIRFLOW-5292] Allow ECSOperator to tag tasks (#5891)
+- [AIRFLOW-4939] Simplify Code for Default Task Retries (#6233)
+- [AIRFLOW-5126] Read ``aws_session_token`` in extra_config of the aws hook (#6303)
+- [AIRFLOW-5636] Allow adding or overriding existing Operator Links (#6302)
+- [AIRFLOW-4965] Handle quote exceptions in GCP AI operators (v1.10) (#6304)
+- [AIRFLOW-3783] Speed up Redshift to S3 UNload with HEADERs (#6309)
+- [AIRFLOW-3388] Add support to Array Jobs for AWS Batch Operator (#6153)
+- [AIRFLOW-4574] add option to provide private_key in SSHHook (#6104)
+- [AIRFLOW-5530] Fix typo in AWS SQS sensors (#6012)
+- [AIRFLOW-5445] Reduce the required resources for the Kubernetes's sidecar (#6062)
+- [AIRFLOW-5443] Use alpine image in Kubernetes's sidecar (#6059)
+- [AIRFLOW-5344] Add --proxy-user parameter to SparkSubmitOperator (#5948)
+- [AIRFLOW-3888] HA for Hive metastore connection (#4708)
+- [AIRFLOW-5269] Reuse session in Scheduler Job from health endpoint (#5873)
+- [AIRFLOW-5153] Option to force delete non-empty BQ datasets (#5768)
+- [AIRFLOW-4443] Document LatestOnly behavior for external trigger (#5214)
+- [AIRFLOW-2891] Make DockerOperator container_name be templateable (#5696)
+- [AIRFLOW-2891] allow configurable docker_operator container name (#5689)
+- [AIRFLOW-4285] Update task dependency context definition and usage (#5079)
+- [AIRFLOW-5142] Fixed flaky Cassandra test (#5758)
+- [AIRFLOW-5218] Less polling of AWS Batch job status (#5825)
+- [AIRFLOW-4956] Fix LocalTaskJob heartbeat log spamming (#5589)
+- [AIRFLOW-3160] Load latest_dagruns asynchronously on home page (#5339)
+- [AIRFLOW-5560] Allow no confirmation on reset dags in `airflow backfill` command (#6195)
+- [AIRFLOW-5280] conn: Remove aws_default's default region name (#5879)
+- [AIRFLOW-5528] end_of_log_mark should not be a log record (#6159)
+- [AIRFLOW-5526] Update docs configuration due to migration of GCP docs (#6154)
+- [AIRFLOW-4835] Refactor operator render_template (#5461)
+
+Bug Fixes
+"""""""""
+- [AIRFLOW-5459] Use a dynamic tmp location in Dataflow operator
```

Comment:
```suggestion
- [AIRFLOW-5459] Use a dynamic tmp location in Dataflow operator (#6078)
```

Reply:
tbh we shouldn't have to do this manually !! So I don't mind if we don't have it

Reply:
I guess this is to do with how it got merged.

Reply:
Yup it is